assets/section-main-product.css

```diff
@@ -369,9 +355,35 @@ a.product__text {
   display: none;
 }
 
+.different-color-scheme {
+  padding: 0;
+}
```

Comment:
There is actually an issue with the quick add modal when the color scheme are the same.

Have a look at [this video](https://screenshot.click/07-37-qte21-z3alg.mp4)

Reply:
Okay. Since it's quite urgent now I assume that we need need to pass color scheme styling to quick-add modal. I didn't find any solution to do that except using JS. Now it seems working good and looks good too.

@Oliviammarcello please confirm rather we need this solution or not. thanks.

Reply:
What if we don't add the colour scheme to the quick add modal and force it to take the colour of the site background. I am not confident that merchants would want an instance where the product information is a different colour.

Reply:
I pushed updates that prevent quick add modal from pulling the styling from PDP.